vagrant/README.md

```diff
@@ -116,26 +171,109 @@ default   backend    service=backend version=v0   Online   6m3s
 default   frontend   service=frontend             Online   2m46s                2m44s              3               0
 default   kong       service=kong                 Online   53s                  52s                3               0
 ```
-
 There are 5 dataplanes which correlates with each component of our application.
 
-### 8. You can also use `kumactl` to look at the mesh. As shown below, our default mesh does not have mTLS enabled.
+### GUI
+
+Kuma ships with an internal GUI that will help you visualize the mesh and its policies in an intuitive format. It can be found on port `:5683` on the control-plane machine. Since our Kuma control-plane machine's IP is `192.168.33.10`, navigate to [http://192.168.33.10:5683/](http://192.168.33.10:5683/) to use Kuma's GUI.
+
+
+## Integrations
+
+### Kong Gateway
+
+In this Vagrant demo, Kong is already configured and deployed when you ran `vagrant up` in the [vagrant setup step](#vagrant). But to give you an in depth dive on how it works, lets dig into the files in the [`/vagrant/kong/`](/vagrant/kong/) directory.
```

Comment:
Why all `/vagrant/...` paths are absolute ? It's confusing.

Let's change `[/vagrant/kong/](/vagrant/kong/)` to `[kong/](./kong/)`